app/models/collection_object.rb

```diff
@@ -562,10 +562,18 @@ def self.from_collecting_events(collecting_event_ids, area_object_ids, area_set,
   # @return [Scope] of selected collection objects through collecting events with georeferences, remember to scope to project!
   def self.in_date_range(search_start_date: nil, search_end_date: nil, partial_overlap: 'on')
     allow_partial = (partial_overlap.downcase == 'off' ? false : true) # TODO: Just get the correct values from the form!
-    where_sql = CollectingEvent.date_sql_from_dates(search_start_date, search_end_date, allow_partial)
+    where_sql     = CollectingEvent.date_sql_from_dates(search_start_date, search_end_date, allow_partial)
     joins(:collecting_event).where(where_sql)
   end
 
+  def self.id_group(namespace, id_type = 'Identifier::Local::CatalogNumber')
```

Comment:
@TuckerJD I'm going to pull in the code, but this method needs to be refactored once in to native SQL.